universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java

```diff
@@ -88,7 +88,10 @@ private ContainerTag branding() {
         .with(
             div()
                 .withId("brand-id")
-                .withClasses(ApplicantStyles.CIVIFORM_LOGO)
+                .withClasses(
+                    ApplicantStyles.CIVIFORM_LOGO,
+                    Styles.OPACITY_75,
+                    StyleUtils.hover(Styles.OPACITY_100))
```

Comment:
Any reason you didn't add these to `CIVIFORM_LOGO` instead?

Reply:
No I wasn't reading and I assume it was a reference class. Fixing!